XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -317,6 +318,11 @@ private void Build()
                         dialog.ShowDialog(Program.MainWindow);
                     }
                 }
+                if (isHost || isPool)

```

Comment:
The page should be available only if the user connects to servers that have the NRPE plugin installed. XC is allowed to connect to older servers which do not have the plugin, so we need to somehow check and hide the page.